Src/Plugins/Security/SEC101.SecurePlaintextSecrets.json

```diff
@@ -46,6 +46,12 @@
           "ContentsRegex": "$SEC101/006.GitHubPat",
           "MessageArguments": { "secretKind": "GitHub personal access token (PAT)" }
         },
+        {
+          "Id": "SEC101/006",
+          "Name": "DoNotExposePlaintextSecrets/GitHubPat",
+          "ContentsRegex": "$SEC101/006.GitHubPat_2021_04_05",
+          "MessageArguments": { "secretKind": "GitHub personal access token (2021-04-05 version)" }
```

Comment:
>GitHub personal access token (2021-04-05 version [](start = 47, length = 48)

just change this to 'GitHub personal access token'

Reply:
Done.

---
In reply to: [608157843](https://github.com/microsoft/sarif-pattern-matcher/pull/366#discussion_r608157843) [](ancestors = 608157843)